lib/common/util.js

```diff
@@ -380,6 +380,14 @@ function makeAuthorizedRequest(config) {
 
   var authorize = gsa(config);
 
+  if (config.customApi) {
```

Comment:
put the `var authorize = gsa(config);` at L381 in the else block of this if. Yes, that means declaring `var authorize;` but also means the call to `gsa` can't mess things up because its called every time unnecessarily